### PR TITLE
add questions app, models and admin

### PR DESCRIPTION
**Pull request by rafalploszanski**

### CODE REVIEW

**Review by bartoszbaran2 — COMMENTED** [review]
new line at the ends!

File: backend_core/questions/models.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+from django.db import models
+
+
+class Question(models.Model):
```
Comment (martynawitkowska) [inline backend_core/questions/models.py:4]: rename field name to singular:
question
answer
technology

**Review by bartoszbaran2 — APPROVED** [review]
good

**Review by martynawitkowska — CHANGES_REQUESTED** [review]
Add migrations

File: backend_core/questions/admin.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
+from django.contrib import admin
+
+from . import models
+
+
+class QuestionAdmin(admin.ModelAdmin):
+    fields = ('question', 'answer', 'technologies')
```
Comment (martynawitkowska) [inline backend_core/questions/admin.py]: technology please :)

**Review by martynawitkowska — APPROVED** [review]

**Pull request merged**